src/test_driver/openiotsdk/integration-tests/lock-app/test_app.py

```diff
@@ -115,7 +115,7 @@ def test_lock_ctrl(device, controller):
                                      userStatus=DoorLock.Enums.UserStatusEnum.kOccupiedEnabled,
                                      userType=DoorLock.Enums.UserTypeEnum.kUnrestrictedUser,
                                      credentialRule=DoorLock.Enums.CredentialRuleEnum.kSingle),
-                                requestTimeoutMs=1000)
+                                requestTimeoutMs=2000)
```

Comment:
I'm a bit surprised that we need so much time to handle the command

Reply:
I removed the commit